### DISCUSSION TITLE
hide content from quire epub output

### DISCUSSION THREAD

**Discussion by 1000camels** [thegetty/quire#725] [Get Help] [answered]
I am trying to figure out how to exclude some content from my epub.
The issue I am having is that most of my classes are gone, so I cannot specific the element to hide.

What am I missing about how quire epub works?

I am still work on v0.

**Comment by geealbers** [comment, marked as answer]
Yes, v0 uses Pandoc to generate EPUB files and Pandoc aggressively strips out a lot of classes and other markup. We've dumped Pandoc in v1 for that largely that reason and now have much more direct and fine-grained control over the EPUB output.

That said, in v0, I believe you should be able to add the class `remove-from-epub` to any element, and it will be stripped out of the build.

**Comment by 1000camels** [comment]
Oh thanks. I didn’t understand what remove-from-epub. I guess it is
actually doing the opposite.
